test/junit/scala/StringContextTest.scala

```diff
@@ -224,11 +224,7 @@ class StringContextTest {
 
       // literals and arg indexes
       f"%%" -> "%",
-      f" mind%n------%nmatter%n" ->
-       """| mind
-          |------
-          |matter
-          |""".stripMargin,
+      f" mind%n------%nmatter%n" -> String.format(" mind%n------%nmatter%n"),
```

Comment:
I thought about something similar but then I realized that this would somewhat weaken the test - i.e. when run on Windows, it would not assert that "%n" gets converted to "\r\n", as it should. So I decided to only adjust the "expected" side of the assertion.